mantidproject/mantid
# Pearl Powder Diffraction Calibration System Test

**Description of work.**

Calibration system test for the new pearl script, this script which is ran within this system test triggers the `pearl_createCal`, `PEARL_createvan` & `PEARL_creategroup` etc, which is usually used at start of cycle by the `Pearl` scientists to generated calibration files, according to Craig Bull. 

**To test**
- Run the test locally (remember to rerun cmake/build system tests to get the files) 
- Make sure all generated files are cleared upon completion of system test
- All builds pass
- Files have been relocated for `PEARL_Focus` 



Fixes #15983.

Does not need to be in the release notes.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
